apps/syntrest/src/main/java/no/nav/registre/syntrest/kubernetes/KubernetesController.java

```diff
@@ -236,11 +231,7 @@ private void waitForDeployment(String appName) throws InterruptedException, ApiE
     }
 
     private String getCorrectTagQuery(String appName) {
-        if (appsFlyttetTilGithub.contains(appName)) {
-            return "query {repository(owner:\"navikt\", name:\"testnorge-syntetiseringspakker\") {packages(names:[\"" + getCorrectGithubPackageName(appName) + "\"] last:1) {nodes {latestVersion{version}} }}}";
-        } else {
-            return "query {repository(owner:\"navikt\", name:\"synt\") {packages(names:[\"" + appName + "\"] last:1) {nodes {latestVersion{version}} }}}";
-        }
+        return "query {repository(owner:\"navikt\", name:\"testnorge-syntetiseringspakker\") {packages(names:[\"" + getCorrectGithubPackageName(appName) + "\"] last:1) {nodes {latestVersion{version}} }}}";
     }
 
     private String getCorrectGithubPackageName(String appName) {
```

Comment:
Funker denne for `synthdata-arena-meldekort`? Der returnerer den vel `synt_meldekort` men det er kanskje riktig? Evt. refaktorere til å matche på `synthdata-`, siden det er det som er likt hver gang?

Reply:
Ja, `synt_meldekort` er korrekt, det er formatet som er brukt for package names i testnorge-syntetiseringspakker. Jeg prøvde først med formatet `synthdata-...` når jeg skulle lagre packages i testnorge-syntetiseringspakker, men github lot meg ikke gjøre det siden synt-repo allerede hadde lagret det på den måten.